src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +47,34 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        DateTimeFormatter formatter = new DateTimeFormatterBuilder()
```

Comment:
you can simplify assigning like this: 
= DateTimeFormatter.ofPattern("dd.MM.yyyy");

Reply:
done